HTML-37/tapnumber.js

```diff
@@ -0,0 +1,72 @@
+let textColorRandomNumber = randomNumberFunction(0,6,6);
+let colors;
+differentTextColor();
+colorTable();
+
+document.getElementById('click-this').addEventListener('click', shuffleRandomTable);
+document.getElementById('table-here').addEventListener('click', selectColors);
+
+function randomNumberFunction(start, end, count) {
+    let returnArray = [],
+        randomNumber;
+    for(let i=0; i<count; i++){
+        randomNumber = Math.floor(Math.random() * (end - start)) + start;
+        if(returnArray.indexOf(randomNumber) == -1){
+            returnArray.push(randomNumber)
+        }else {
+            --i;
+        }
+    }
+    return randomNumber
+}
```

Comment:
Too much redundant code, function can use just 1 param (or two if you want to reuse it).